src/misc/setup.cpp

```diff
@@ -1151,6 +1167,16 @@ bool CommandLine::FindInt(char const * const name,int & value,bool remove) {
 	return true;
 }
 
+bool CommandLine::FindIntOption(const std::string &name, int &value, bool remove) {
+	cmd_it it,it_next;
+	if (!IsOption(name, it))
+		return false;
+	it_next=it;++it_next;
+	value=atoi((*it_next).c_str());
```

Comment:
If `it` is the last valid value, ie `*it == cmds.back()`, then incrementing `++it_next` will place it on the `end()` value, which can't be deferenced.

Suggest another assert that `it_next` isn't `end()` prior to dereferencing.

Reply:
Done.